packages/gatsby/index.d.ts

```diff
@@ -537,7 +537,11 @@ export interface GatsbyBrowser {
     args: ReplaceComponentRendererArgs,
     options: PluginOptions
   ): any
-  replaceHydrateFunction?(args: BrowserPluginArgs, options: PluginOptions): any
+  replaceHydrateFunction?(args: BrowserPluginArgs, options: PluginOptions): (
+    element: React.DOMElement<React.DOMAttributes<Element>, Element>,
+    container: Element,
+    callback: () => {}
+  ) => void
```

Comment:
Because we want the same signature as `hydrate` or `render` from `react-dom` - could we import type of that from `(@types/)react-dom` and use that instead of writing custom one?

Reply:
Best match that I found in `react-dom` is `Renderer` interface. See that would be satisfied.